Runtime/BlockadeDemoCamera.cs

```diff
@@ -212,6 +249,11 @@ private void UpdateAutoPanning()
 
         private bool MouseIsOverUI()
         {
+            if (_rectTransform)
+            {
+                return !RectTransformUtility.RectangleContainsScreenPoint(_rectTransform, Input.mousePosition);
```

Comment:
this seems like a hack, or at least, can we rename the method?

Reply:
suggestions for a better name?